site/index.html

```diff
@@ -119,33 +143,89 @@ <h2>Sample NGS features</h2>
                         </li>
                     </ul>
                 </li>
+                <li>
+                    and much, much more :)
+                </li>
             </ul>
 
-        </div>
-        <div class="col-sm">
-            <h2>Documentation</h2>
+            <h2 class="text-left">Links</h2>
             <ul>
-                <li><a href="doc/latest/index.html">Documentation Index</a> - main entry point to all documentation</li>
+                <li><a href="http://tut.ngs-lang.org" target="_blank">Examples-based tutorial</a> - understand how NGS feels like</li>
+                <li><a href="http://why.ngs-lang.org" target="_blank">Why use NGS? Why it was created?</a></li>
+                <li><a href="http://lang.ngs-lang.org" target="_blank">Documentation: Language description</a></li>
+                <li><a href="http://reference.ngs-lang.org" target="_blank">Documentation: Language reference</a></li>
+                <li><a href="http://style.ngs-lang.org" target="_blank">Documentation: Coding style guide</a></li>
+                <li><a href="http://source.ngs-lang.org" target="_blank">GitHub</a></li>
+                <li><a href="http://discord.ngs-lang.org" target="_blank">Discord</a></li>
+                <li><a href="http://videos.ngs-lang.org" target="_blank">YouTube</a></li>
             </ul>
         </div>
-
-    </div>
-    <div class="row">
+        <div  style="margin-left: 40px"></div>
         <div class="col-sm">
-            <h2>Source</h2>
-            <a href="https://github.com/ngs-lang/ngs">https://github.com/ngs-lang/ngs</a>
+            <h2 class="text-left">Quick Start</h2>
```

Comment:
This section still looks somewhat cluttered.

Reply:
Highlighting is overkill here

Reply:
Removed the section and text. Please check if it looks better. (definitely looks less cluttered, but also may look a bit lost there)